README.md

```diff
@@ -69,12 +69,29 @@ Language: You must have GoLang installed to run Scorecards (https://golang.org/d
 
 ### Installation
 
-To install Scorecards:
+#### Standalone
+
+To install Scorecards as a standalone:
 
 1. Visit our latest [release page](https://github.com/ossf/scorecard/releases/latest) and download the correct binary for your operating system
 2. Extract the binary file 
 3. Add the binary to your `GOPATH/bin` directory (use `go env GOPATH` to identify your directory if necessary)
 
+#### Using Homebrew
+
+You can use [Homebrew](https://brew.sh/) (on macOS or Linux) to install Scorecards.
+
+```sh
+brew install scorecard
```

Comment:
Does this actually work? I thought we don't have homebrew setup. @developer-guy fyi

Reply:
Looks like there is an existing https://formulae.brew.sh/formula/scorecard#default

Reply:
Looks like this is also maintained by the community.

Reply:
I see, in that case do we need #1376?

Reply:
I don't think we need this now. cc  @developer-guy